docs/reference/starting-agent.md

```diff
@@ -168,14 +174,14 @@ Unfortunately, **using a bundler typically breaks the APM agent**. Bundling mult
 1. exclude the `elastic-apm-node` APM agent module from the bundle; and
 2. optionally exclude other modules from the bundle that you would like the APM agent to instrument.
 
-"Excluding" a module *foo* from the bundle (Webpack calls these "externals") means that a `require('foo')` expects "node_modules/foo/…​" to exist at runtime. This means that you need to deploy both your bundle file(s) *and* the excluded modules. This may or may not defeat your reasons for using a bundler.
+"Excluding" a module *foo* from the bundle (Webpack calls these "externals") means that a `require('foo')` expects "node_modules/foo/… " to exist at runtime. This means that you need to deploy both your bundle file(s) *and* the excluded modules. This may or may not defeat your reasons for using a bundler.
```

Comment:
```suggestion
"Excluding" a module *foo* from the bundle (Webpack calls these "externals") means that a `require('foo')` expects "node_modules/foo/…" to exist at runtime. This means that you need to deploy both your bundle file(s) *and* the excluded modules. This may or may not defeat your reasons for using a bundler.
```